api/migrations/versions/0238_add_sap_sms_provider.py

```diff
@@ -0,0 +1,45 @@
+"""
+
+Revision ID: 0238
+Revises: 0237
+Create Date: 2019-11-29 13:43:56.438399
+
+"""
+from alembic import op
+from datetime import datetime
+import uuid
+
+
+revision = '0238'
+down_revision = '0237'
+
+
+def upgrade():
+    op.execute(
+        "INSERT INTO provider_details (id, display_name, identifier, priority, notification_type, active, supports_international, version) values ('{}', 'SAP', 'sap', 20, 'sms', true, true, 1)".format(str(uuid.uuid4()))
+    )
+
+    op.execute((
+        "INSERT INTO provider_rates (id, valid_from, rate, provider_id) VALUES ('{}', '{}', 4.0, "
```

Comment:
we want this as 6.0

Reply:
updated locally. won't fiddle with the branch or DB migrations for this one, so expect the price is wrong in this feature env